.ado/jobs/integration-test.yml

```diff
@@ -57,11 +57,6 @@ jobs:
         displayName: yarn windows --no-launch
```

Comment:
you'll want to revise the e2e test yml file since it's integrated with uploading the masters and the outputs